gladier_xpcs/tools/xpcs_boost_corr.py

```diff
@@ -57,7 +74,7 @@ class BoostCorr(GladierBaseTool):
 
 if __name__ == '__main__':
     data = {
-        'proc_dir':'/eagle/APSDataAnalysis/nick/xpcs_gpu',
+        'proc_dir':'/eagle/APSDataAnalysis/nick/xpcs_gpu/C032_B315_A200_150C_att01_001_0001-1000',
```

Comment:
Can we move that to `/XPCS/test_boost_corr/` where everyone have access?

Reply:
Previously that didn't work due to fs permissions -- Only the first person who ran would be successful, then their files would be set without group writable perms and the second run by another person would fail.